exporter/command.go

```diff
@@ -76,6 +76,7 @@ func (ic *importContext) interactivePrompts() {
 // Run import according to flags
 func Run(args ...string) error {
 	log.SetOutput(&logLevel)
+	log.Printf("[WARN] This tooling is experimental and provided as is. It has an evolving interface, which may change or be removed in future versions of the provider.")
```

Comment:
I'm not sure that adding this to the code is required... Exporter functionality isn't something that people just find in the docs, and it's already covered in the docs. Otherwise we'll need to add such warnings to most of our tooling - CLI, SDKs, etc.

Reply:
We want to highlight that it is experimental so it would also be nice to have this in the code.